pkg/util/kubernetes/kubelet/podwatcher.go

```diff
@@ -65,7 +67,9 @@ func (w *PodWatcher) PullChanges() ([]*Pod, error) {
 	if err != nil {
 		return podList, err
 	}
-	return w.computeChanges(podList)
+	changes, err := w.computeChanges(podList)
+
+	return changes, err
```

Comment:
out of curiosity, what's the rationale behind this change?

Reply:
Oh, this is a leftover from my debugging. Thanks for the catch. Changing it 👍